README.md

```diff
@@ -156,10 +156,12 @@ The special E2E client settings should be defined in `env.yml`:
 | AWS_API_SERVER | string | Optional | aws-api-id.execute-api.us-east-1.amazonaws.com | For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server. |
 | AWS_API_STAGE | string | Optional | stage-name | For generating the swagger document at runtime. Can be null, however that will prevent tools such as swagger-UI from submitting test requests to the API server. |
 | BUGSNAG_API_KEY | string | Required | 123e4567e89b12d3a4564266 | A valid Bugsnag authorization token. |
+| BUGSNAG_APP_TYPE | string | Optional | my-application | An identifier of the app type of this otp-middleware instance. This can be used to segment errors within the Bugsnag dashboard. |
```

Comment:
Would you like to put examples as you did in your technical doc about this?

Reply:
I think this is technical enough information that if someone wants to enter in this information they can do some additional research to figure out what this means and if they want to set these values to something.